frontend_app_kanban/src/component/KanbanCard.jsx

```diff
@@ -140,6 +141,24 @@ function KanbanCard (props) {
           />
         )}
       </div>
+      <div
+        className='kanban__contentpage__wrapper__board__card__options'
+      >
+        {props.card.deadline !== '' && (
+          <div
+            className='deadline'
```

Comment:
Just a suggestion, if you don't like, don't need to apply

```suggestion
            className='kanban__contentpage__wrapper__board__card__options__deadline'
```